src/fsharp/FSharp.Build/Microsoft.FSharp.Overrides.NetSdk.targets

```diff
@@ -0,0 +1,36 @@
+<!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information. -->
+
+<!--
+***********************************************************************************************
+Microsoft.FSharp.Overrides.NetSdk.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <Target Name="CoreGenerateAssemblyInfo"
+          Condition="'$(Language)'=='F#'"
```

Comment:
Update the condition to `'$(Language)' == 'F#' AND '$(GenerateAssemblyInfo)' != 'false'`.

Reply:
I believe this is handled by the DotNET Sdk  here: https://github.com/dotnet/sdk/blob/6945e3694c918eea4c8c4fb6217e1485b179994b/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAssemblyInfo.targets